client/src/app/tabs/bpmn/custom/properties-provider/helper/InputOutputHelper.js

```diff
@@ -125,14 +123,17 @@ export function areInputParametersSupported(element) {
    * @return {boolean} a boolean value
    */
 export function areOutputParametersSupported(element) {
-  const messageEventDefinition = eventDefinitionHelper.getMessageEventDefinition(element);
-
   return isAny(element, [
     'bpmn:ServiceTask',
     'bpmn:SubProcess',
     'bpmn:ReceiveTask',
-    'bpmn:CallActivity'
-  ]) || messageEventDefinition;
+    'bpmn:CallActivity',
+    'bpmn:StartEvent',
+    'bpmn:EndEvent',
+    'bpmn:IntermediateThrowEvent',
+    'bpmn:IntermediateCatchEvent',
+    'bpmn:BoundaryEvent'
```

Comment:
How about:
```suggestion
    'bpmn:Event'
```
instead?

Reply:
:exploding_head:

Reply:
That would reflect this requirement explicitly:

> Zeebe is supporting output mappings for all events already.

Reply:
I missed your response, sorry 😅